rust/Cargo.toml

```diff
@@ -11,10 +11,24 @@ homepage = "https://roualdes.github.io/bridgestan/latest/"
 [dependencies]
 libloading = "0.8.0"
 thiserror = "1.0.40"
+ureq = { version = "2.7", optional = true }
+tar = { version = "0.4", optional = true }
+flate2 = { version = "1.0", optional = true }
+dirs = { version = "5.0", optional = true }
+path-absolutize = { version = "3.1", optional = true }
+log = { version = "0.4", optional = true }
+
+[features]
+compile-stan-model = ["ureq", "tar", "flate2", "dirs", "path-absolutize", "log"]
```

Comment:
I think we should not expose the optional dependencies as features, ie use the "dep:cratename" syntax mentioned here: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
well spotted, thanks! I've read this part of the Cargo book more than once but forgot about the implicit feature creation